Airwallex/Payment/Sources/ApplePaySectionController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ApplePaySectionController.swift
+//  Airwallex
+//
+//  Created by Weiping Li on 2024/12/19.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+class ApplePaySectionController: SectionController {
+    
+    var context: CollectionViewContext<PaymentSectionType, String>!
+
+    let section = PaymentSectionType.applePay
+    private(set) var session: AWXSession
```

Comment:
If it's initialised in contructor only, a `let` property might be more concise and constrain its usage.

Reply:
addressed